From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Naor Livne <naorlivne@gmail.com>
Date: Fri, 1 Jan 2021 14:31:14 +0200
Subject: [PATCH] bump idna has version 3.0 was released

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 7ba4b2a..f265384 100755
--- a/setup.py
+++ b/setup.py
@@ -43,7 +43,7 @@ packages = ['requests']
 
 requires = [
     'chardet>=3.0.2,<5',
-    'idna>=2.5,<3',
+    'idna>=2.5,<4',
     'urllib3>=1.21.1,<1.27',
     'certifi>=2017.4.17'
 

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Naor Livne <naorlivne@gmail.com>
Date: Tue, 5 Jan 2021 16:31:15 +0200
Subject: [PATCH] Update setup.py
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Co-authored-by: Mickaël Schoentgen <contact@tiger-222.fr>
---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index f265384..5ce59e6 100755
--- a/setup.py
+++ b/setup.py
@@ -43,7 +43,8 @@ packages = ['requests']
 
 requires = [
     'chardet>=3.0.2,<5',
-    'idna>=2.5,<4',
+    'idna>=2.5,<3 ; python_version < "3"',
+    'idna>=2.5,<4 ; python_version >= "3"',
     'urllib3>=1.21.1,<1.27',
     'certifi>=2017.4.17'
 
